pkg/notes/options.go

```diff
@@ -143,6 +152,27 @@ func (o *Options) ValidateAndFinish() error {
 		}
 	}
 
+	// Fail early if the template is broken.
+	if o.Format == FormatOutputMarkdown && o.TemplatePath != "" {
+		// Look in default directory if using default template.
+		if o.TemplatePath == DefaultMarkdownTemplate {
```

Comment:
I think that's strange. Why only search in the templates directory when this "random magic" template name is provided?
Let's just always assume the user needs to provide a proper path to a file and don't do any strange magic here?

Reply:
Makes sense. I'll remove the magic :)

Reply:
Removed.